web/src/components/ShelterInformation.tsx

```diff
@@ -24,6 +24,10 @@ import ShelterInformationSection from './ShelterInformationSection'
 import { StyledButton } from './TextButton'
 import Tooltip from './Tooltip'
 
+const FullWidth = styled.div`
+  flex: 1;
```

Comment:
so `flex: 1` in container (line 27) is not needed any longer.

Reply:
Removed flex in line 32.